internal/service/base/resource_language_translation_test.go

```diff
@@ -0,0 +1,269 @@
+// Copyright © 2025 Ping Identity Corporation
+// Code generated by ping-terraform-plugin-framework-generator
+
+package base_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingone/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingone/internal/acctest/service/base"
+	client "github.com/pingidentity/terraform-provider-pingone/internal/client"
+)
+
+func TestAccLanguageTranslation_RemovalDrift(t *testing.T) {
+	t.Parallel()
+
+	resourceName := acctest.ResourceNameGen()
+	resourceFullName := fmt.Sprintf("pingone_language_translation.%s", resourceName)
+
+	environmentName := acctest.ResourceNameGenEnvironment()
+
+	licenseID := os.Getenv("PINGONE_LICENSE_ID")
+	var environmentId string
+	var locale string
+
+	var p1Client *client.Client
+	var ctx = context.Background()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			acctest.PreCheckClient(t)
+			acctest.PreCheckNewEnvironment(t)
+			acctest.PreCheckNoFeatureFlag(t)
+
+			p1Client = acctest.PreCheckTestClient(ctx, t)
+		},
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			// Test removal of the environment
+			{
+				Config: languageTranslation_NewEnvHCL(environmentName, licenseID, resourceName),
+				Check:  languageTranslation_GetIDs(resourceFullName, &environmentId, &locale),
+			},
```

Comment:
We'll want to cover removal of the translation too here. The singleton test template doesn't include that which is probably why it is missing. You can look at resource_custom_role_test.go for an example for a non-singleton